### PR TITLE
changes default prefs to deadmin you as antagonist/head/sec/silicon, stops auto deadmin on localhost

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request
the default prefs for auto deadmin are now true for antag, sec, silicon, head
localhost stops auto deadmin from working

## Why It's Good For The Game
on the tgstation servers we have turned auto deadmin off, but the default preference should definitely be to auto deadmin you, with it being opt out for people that like the risk :D
on localhost you dont want to lose admin when debugging

## Changelog
:cl:
admin: you no longer get auto deadminned on localhost
admin: default prefs are now to be deadminned as antag/head/sec/silicon
/:cl:

### CODE REVIEW

**Comment by ZephyrTFA** [comment]
@tgstation/head-admins

**Comment by DrAmazing343** [comment]
We've +1'd this PR. Admins ought to be allowed to change preferences on their own without us forcing a rather arbitrary auto-deadmin on them. I trust our boys on this one, and in any case, it was already very easy to re-admin but just very annoying before.

File: code/__DEFINES/preferences.dm [hunk 1]
```diff
@@ -17,7 +17,8 @@
 #define ADMIN_IGNORE_CULT_GHOST (1<<21)
 #define SPLIT_ADMIN_TABS (1<<23)
 
-#define TOGGLES_DEFAULT (SOUND_ADMINHELP|MEMBER_PUBLIC|SOUND_PRAYERS)
+#define TOGGLES_DEADMIN_DEFAULT (DEADMIN_ANTAGONIST|DEADMIN_POSITION_HEAD|DEADMIN_POSITION_SECURITY|DEADMIN_POSITION_SILICON)
```
Comment (SmArtKar) [inline code/__DEFINES/preferences.dm:20]: You're... not using this anywhere? Have you forgotten to commit a file or smth?

**Review by SmArtKar — CHANGES_REQUESTED** [review]

File: code/__DEFINES/preferences.dm (hunk 1 above)
Comment (Fikou) [reply code/__DEFINES/preferences.dm:20]: im literally using it in the line below

Comment (SmArtKar) [reply code/__DEFINES/preferences.dm:20]: ARGH firefox i hate your f5

**Pull request merged**